users/permissions.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import permissions
+
+class IsModerator(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+        return request.user and request.user.is_authenticated and \
```

Comment:
Достаточно одной проверки
```suggestion
        return request.user.groups.filter(name='moderators').exists()
```